common/src/cellular_at_core.c

```diff
@@ -68,11 +68,6 @@ static CellularATError_t validateString( const char * pString,
     CellularATError_t atStatus = CELLULAR_AT_SUCCESS;
     size_t stringLength = 0U;
 
-    if( ( pString == NULL ) || ( pStringValidationResult == NULL ) )
```

Comment:
With this change, validateString will only ever return CELLULAR_AT_SUCCESS. Is it necessary to pStringValidationResult via a pointer or could the validateString function just return pStringValidationResult?

Reply:
Yes, this change makes the CELLULAR_AT_SUCCESS meaningless.
Let me fix this changes.